front/tests/assets/operationStudies/test_variables.ts

```diff
@@ -8,4 +8,11 @@ export default {
     ? 'rollingstock-_@Test BB 7200GVLOCOMOTIVES'
     : 'rollingstock-1TGV2N2',
   rollingStockInfo: process.env.CI ? /BB 7200GV/ : /TGV2N2US/,
+  originSearch: process.env.CI ? 'south west' : 'beaune',
```

Comment:
It would be cleaner to have something like this:
```
  export default process.env.CI
    ? { infraName: 'small_infra', ... }
    : { infraName: 'France', ... };
```

Reply:
Done